documentation/configuration.md

```diff
@@ -21,5 +21,19 @@ metrics: {
             enabled: true
         }
     }
+
+    jolokia: {
```

Comment:
@Sharpie yeah, I think you're totally right about that. I do wonder if there's ever any other config that we might want around the metrics webservice other than just jolokia? But also given that there's already other edits/API breakage we might end up making in the future, I'm good with it like this.

I guess there is one consideration though - in pe-tk-metrics you have to say which reporters you want for each registry. Should we do something similar for this? Or do we say that the metrics webservice type things are only enable/disable-able globally, given that they're turned on by a separate service? I think I'm okay saying the latter. I think I might be leaning towards having a separate `metrics-webservice` config section though. Thoughts?